CONTRIBUTING.md

```diff
@@ -0,0 +1,118 @@
+# Contributing to Generic Node Sensor Edition
+
+Thank you for your interest in helping us build the Generic Node Sensor Edition (GNSE).
+
+We provide the hardware and firmware of the device to support the LoRaWAN ecosystem and help build awesome LoRaWAN devices.
+Any contribution to the project is appreciated.
+
+There are many ways to contribute to this project.
+
+## Types of contributions :memo:
+
+### :mega: Discussions
+
+We love to discuss design decisions, device features and possible use cases with the community, so please use the [forum](https://www.thethingsnetwork.org/forum/). We have a special [category for The Generic Node](https://www.thethingsnetwork.org/forum/c/nodes/generic-node/88).
+
+### :question: Questions
+
+ If you have questions, please use the [forum](https://www.thethingsnetwork.org/forum/). We have a special [category for The Generic Node](https://www.thethingsnetwork.org/forum/c/nodes/generic-node/88).
+
+### :bulb: New features
+
+ If you have a great idea or think some functionality is missing, we want to know! The only thing you have to do for that is to [create an issue](https://github.com/TheThingsIndustries/generic-node-se/issues) if it doesn't exist yet. Please use the issue template and fill out all sections.
+
+
+### :beetle: Issues
+
+If you notice that one of the applications is not functioning properly or there may be a bug in the implementation. In this case you should [create an issue](https://github.com/TheThingsIndustries/generic-node-se/issues) if it doesn't exist yet. Please use the issue template and fill out all sections.
+
+### :hammer_and_wrench: Pull requests
+
+If you see an [open issue](https://github.com/TheThingsIndustries/generic-node-se/issues) that you would like to work on, let the other contributors know by commenting in the issue.
+
+### :books: Documentation
+
+If you see that our documentation is lacking or incorrect, it would be great if you could help us improve it. This will help users and fellow contributors understand how to better develop and use the device. Generic Node documentation [repository](https://github.com/TheThingsIndustries/generic-node-docs) can be viewed at [documentation website](https://thethingsindustries.github.io/generic-node-docs) and you can contribute by following the [documentation contribution guideline](https://github.com/TheThingsIndustries/generic-node-docs/blob/master/CONTRIBUTING.md).
```

Comment:
```suggestion
If you see that our documentation is lacking or incorrect, it would be great if you could help us improve it. This will help users and fellow contributors understand how to better develop and use the device. Generic Node documentation that can be viewed on the [documentation website](https://thethingsindustries.github.io/generic-node-docs) is built from [this repository](https://github.com/TheThingsIndustries/generic-node-docs) and you can contribute to it by following the [documentation contribution guideline](https://github.com/TheThingsIndustries/generic-node-docs/blob/master/CONTRIBUTING.md).
```